tests/scripts/thread-cert/Cert_5_2_06_RouterDowngrade.py

```diff
@@ -227,7 +227,7 @@ def test(self):
         self.simulator.go(config.ROUTER_STARTUP_DELAY)
         self.assertEqual(self.nodes[ROUTER23].get_state(), 'router')
 
-        self.simulator.go(10)
+        self.simulator.go(config.LEADER_STARTUP_DELAY)
```

Comment:
Why `LEADER_STARTUP_DELAY` here?

Reply:
Thanks Jonathan for going through it all and finding all cases. 😃

Reply:
Thanks Jonathan. I tried to search in code to make sure all uses `LEADER_STARTUP_DELAY` are followed by state check for some node becoming leader. Used `git grep -A 1 "LEADER_STARTUP_DELAY"` then parsed the output to make sure the line after has the state check. Found a bunch of other instances of it being used incorrectly. Hopefully managed to find them all